connect/runtime/src/test/java/org/apache/kafka/connect/runtime/ConnectorConfigTest.java

```diff
@@ -69,7 +69,7 @@ public void close() {
         @Override
         public ConfigDef config() {
             return new ConfigDef()
-                    .define("magic.number", ConfigDef.Type.INT, ConfigDef.NO_DEFAULT_VALUE, ConfigDef.Range.atLeast(42), ConfigDef.Importance.HIGH, "");
+                .define("magic.number", ConfigDef.Type.INT, ConfigDef.NO_DEFAULT_VALUE, ConfigDef.Range.atLeast(42), ConfigDef.Importance.HIGH, "");
```

Comment:
Minor nit: it'd be good to not include unnecessary changes like this.

Reply:
That was one of those cases of Intellj meaning well and me not catching it, will revert.